From 2fc8501e44e98b50b6344f466a4536e4987a69c2 Mon Sep 17 00:00:00 2001
From: Tao Huang <huangtao@rock-chips.com>
Date: Thu, 26 Jul 2018 15:17:59 +0800
Subject: [PATCH] Revert "ARM: dma-mapping: silence warning if
 atomic_pool_size=0"

This reverts commit 234d4214216483b28eb9c2df07944c3adb3cb70b.
We do not use coherent_pool=0 anymore.

Change-Id: I3b05f15955c8f4b156cf8a4dfd8c0977e33a24b9
Signed-off-by: Tao Huang <huangtao@rock-chips.com>
---
 arch/arm/mm/dma-mapping.c | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/arch/arm/mm/dma-mapping.c b/arch/arm/mm/dma-mapping.c
index ea4b5ef7f287..d539dee3c78d 100644
--- a/arch/arm/mm/dma-mapping.c
+++ b/arch/arm/mm/dma-mapping.c
@@ -338,9 +338,6 @@ static int __init atomic_pool_init(void)
 	struct page *page;
 	void *ptr;
 
-	if (!atomic_pool_size)
-		return 0;
-
 	atomic_pool = gen_pool_create(PAGE_SHIFT, -1);
 	if (!atomic_pool)
 		goto out;
-- 
2.35.3

